Trie/Trie/Trie/Trie.swift

```diff
@@ -62,11 +60,32 @@ class Trie {
     fileprivate let root: Node
     fileprivate var wordCount: Int
 
-    /// Creats an empty trie.
-    init() {
+    /// Creates an empty trie.
+    override init() {
```

Comment:
You bring up a good point, I'm not sure why the compiler didn't complain in this situation, but according to the `NSObject` [docs](https://developer.apple.com/reference/objectivec/nsobject/1418641-init): 

> In a custom implementation of this method, you must invoke super’s Initialization then initialize and return the new object.

Personally speaking, I always call `super` whenever I override anything. Haven't come across a situation where not calling `super` provided any benefits. Let me know if you have another perspective on this.